srcpkgs/lazyorg/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'lazyorg'
+pkgname=lazyorg
+version=1.1.1
+revision=1
+build_style=go
+go_import_path=github.com/HubertBel/lazyorg
+go_package="./cmd/lazyorg"
+short_desc="Simple terminal-based calendar and note-taking application"
+maintainer="Xavier Fortier <xavier.fortier@proton.me>"
+license="MIT"
+homepage="https://github.com/HubertBel/lazyorg"
```

Comment:
```suggestion
homepage="https://github.com/HubertBel/lazyorg"
changelog="https://github.com/HubertBel/lazyorg/releases"
```

Reply:
Would it be better to put this to get the changelog of the particular tag?
```sh
changelog=https://github.com/HubertBel/lazyorg/releases/tag/v{version}
```